cli/src/program.rs

```diff
@@ -198,6 +219,19 @@ impl ProgramSubCommands for App<'_, '_> {
                                 .long("allow-excessive-deploy-account-balance")
                                 .takes_value(false)
                                 .help("Use the designated program id even if the account already holds a large balance of SOL")
+                        )
+                        .arg(
+                            Arg::with_name("use_quic")
+                                .long("use-quic")
+                                .global(true)
```

Comment:
presumably the point of moving this arg declaration from the app to the subcommand was that it _not_ be global?

Reply:
It was just used in program deploy and write buffer. I will make it not global.